mod/friends_collections/views/default/collections/listing/owner.php

```diff
@@ -7,11 +7,11 @@
  */
 
 $entity = elgg_extract('entity', $vars);
-if (!$entity) {
+if (!($entity instanceof \ElggEntity)) {
```

Comment:
why not `ElggUser`?

Reply:
there are no user entity specific functions used in this view or other checks on the type